src/mongoose_http_handler.erl

```diff
@@ -16,8 +16,9 @@
 
 -callback config_spec() -> mongoose_config_spec:config_section().
 -callback routes(options()) -> routes().
+-callback http_handler_instrumentation() -> [mongoose_instrument:spec()].
 
--optional_callbacks([config_spec/0, routes/1]).
+-optional_callbacks([config_spec/0, routes/1, http_handler_instrumentation/0]).
```

Comment:
1. I would call it `instrumentation` for simplicity and consistency.
2. This module should call the function from the module, not `ejabberd_cowboy`.